apps/music/style/music.css

```diff
@@ -585,10 +585,12 @@ body.picker-mode #title-player {
 
 #player-cover.slideOut #player-cover-caption {
   transform: translateY(100%);
+  visibility: visible;
```

Comment:
Yes, without the visibility, the screen reader will still be able to land on it even though it's hidden from the sighted user.